atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -804,6 +808,8 @@ def _update_extra(self, extra, metadata, worker_metadatas):
         extra['image']['sources_signing_intent'] = source_result['signing_intent']
         extra['osbs_build']['kind'] = KOJI_KIND_IMAGE_SOURCE_BUILD
         extra['osbs_build']['engine'] = KOJI_SOURCE_ENGINE
+        if self.userdata:
+            extra['custom_user_metadata'] = self.userdata
```

Comment:
This is exactly the same for source containers and binary containers, maybe put it in `KojiImportBase.get_build` instead?

Reply:
I don't want to add _update_extra to the base to update just this

Reply:
Ack, I think it could be directly in https://github.com/containerbuildsystem/atomic-reactor/blob/1719a239800ce2c2eba617b870cd298a6a959681/atomic_reactor/plugins/exit_koji_import.py#L388

But it doesn't matter much either way